From 2b0b73b0a1f9564874939f335675091f66c43050 Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Sat, 16 Nov 2024 17:16:01 +0100
Subject: [PATCH] build: fix svg pixbuf loader check with librsvg 2.59+

Since librsvg moved to meson the loader modules are named differently
and building gimp2 on Windows fails to detect them in configure.

Adjust the checks to the new filenames.
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 71b71397ece..6f405ad6766 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2504,8 +2504,8 @@ if test "x$enable_vector_icons" != "xno"; then
     # error: cannot check for file existence when cross compiling
     # So let's test files the shell way.
     if (test "x$platform_win32" = "xyes" &&
-        test -f "$gdk_pixbuf_moduledir/libpixbufloader-svg.dll") ||
-       test -f "$gdk_pixbuf_moduledir/libpixbufloader-svg.so"; then
+        test -f "$gdk_pixbuf_moduledir/pixbufloader_svg.dll") ||
+       test -f "$gdk_pixbuf_moduledir/libpixbufloader_svg.so"; then
       # We must not use $PKG_CONFIG nor PKG_CHECK_* macros because we need
       # to make sure we use the native pkg-config (in case we cross-compile).
       if pkg-config --atleast-version=glib_required_version glib-2.0 &&
-- 
GitLab

